support/cas-server-support-saml/src/test/java/org/apereo/cas/support/saml/util/Saml20ObjectBuilderTests.java

```diff
@@ -0,0 +1,44 @@
+package org.apereo.cas.support.saml.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Tests the {@link AbstractSaml20ObjectBuilder}.
+ *
+ * @author Jerome Leleu
+ * @since 5.2.9
+ */
+public class Saml20ObjectBuilderTests {
```

Comment:
I suspect this should be added to the test suite for the project.

Reply:
Yes. Will do.